lib/sandboxed_renderer/init.js

```diff
@@ -80,16 +92,15 @@ if (window.location.protocol === 'chrome-devtools:') {
 // and any `require('electron')` calls in `preload.js` will work as expected
 // since browserify won't try to include `electron` in the bundle, falling back
 // to the `preloadRequire` function above.
-if (preloadPath) {
-  const preloadSrc = fs.readFileSync(preloadPath).toString()
-  const preloadWrapperSrc = `(function(require, process, Buffer, global, setImmediate) {
+if (preloadSrc) {
```

Comment:
you need to check here if preloadSrc is an Error

Reply:
you are not logging the error here

Reply:
I've used  `preloadError` for the error